datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -105,7 +105,7 @@ export default function UserProfile() {
             (data?.corpUser && entityRegistry.getDisplayName(EntityType.CorpUser, data?.corpUser)) ||
             undefined,
         role: data?.corpUser?.editableProperties?.title || data?.corpUser?.info?.title || undefined,
-        team: data?.corpUser?.editableProperties?.teams?.join(',') || undefined,
+        team: data?.corpUser?.info?.departmentName || data?.corpUser?.editableProperties?.teams?.join(',') || undefined,
```

Comment:
This is a breaking experience change for all existing datahub users. Please use team first, THEN fallback to department name.

Reply:
done